elftools/elf/descriptions.py

```diff
@@ -177,9 +177,12 @@ def describe_note(x):
     n_desc = x['n_desc']
     desc = ''
     if x['n_type'] == 'NT_GNU_ABI_TAG':
-        desc = '\n    OS: %s, ABI: %d.%d.%d' % (
-            _DESCR_NOTE_ABI_TAG_OS.get(n_desc['abi_os'], _unknown),
-            n_desc['abi_major'], n_desc['abi_minor'], n_desc['abi_tiny'])
+        if x['n_name'] == 'Android': # Lame patch :(
```

Comment:
Is this for readelf compatibility?

There's a lot of code like this in `descriptions` so I'd remove the comment.

Reply:
Yes, to keep up with readelf :( Removed.